config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'application#hello'
+  root "static_pages#home"
+  get  "/signup", to: "users#new"
+  get  "/help", to: "static_pages#help"
+  get  "/about", to: "static_pages#about"
+  get  "/contact", to: "static_pages#contact"
```

Comment:
Only one space is allowed between the parameters. 
``
get  "/contact", to: "static_pages#contact"
``